pkg/service/roommanager.go

```diff
@@ -310,6 +310,13 @@ func (r *RoomManager) getOrCreateRoom(ctx context.Context, roomName string) (*rt
 		if err != nil {
 			logger.Errorw("could not handle participant change", err)
 		}
+		// update roomstore with new numParticipants
```

Comment:
maybe this would be unnecessary if in https://github.com/livekit/livekit-server/pull/200 we also call `StoreRoom` after `StoreParticipant` ?

Reply:
I think that would make it more clear - calling `StoreRoom` between `StoreParticipant` and `telemetry.ParticipantJoined`, and then also calling `StoreRoom` between `DeleteParticipant` and `telemetry.ParticipantLeft` in `participant.OnClose`